be/src/exec/pipeline/exchange/local_exchange_sink_operator.cpp

```diff
@@ -24,6 +24,8 @@ Status LocalExchangeSinkOperator::prepare(RuntimeState* state) {
     RETURN_IF_ERROR(Operator::prepare(state));
     _exchanger->incr_sinker();
     _unique_metrics->add_info_string("ShuffleNum", std::to_string(_exchanger->source_dop()));
+    _peak_memory_usage_counter = _common_metrics->AddHighWaterMarkCounter(
+            "PeakMemoryUsage", TUnit::BYTES, RuntimeProfile::Counter::create_strategy(TUnit::BYTES));
```

Comment:
Better to add a prefix, like 'ExchangePeakMemoryUsage' or other(up to you), because the name `PeakMemoryUsage` is very unique used in MemTracker

Reply:
ok